classes/class-wc-connect-stripe.php

```diff
@@ -163,14 +163,24 @@ private function get_default_stripe_config() {
 			return $result;
 		}
 
-		public function maybe_show_banner() {
-			if ( 'yes' !== WC_Connect_Options::get_option( 'banner_stripe', null ) ) {
+		public function maybe_show_notice() {
+			$setting = WC_Connect_Options::get_option( 'banner_stripe', null );
+			if ( is_null( $setting ) ) {
 				return;
 			}
 
-			if ( isset( $_GET[ 'wcs_stripe_code' ] ) ) {
-				$this->connect_oauth( $_GET[ 'wcs_stripe_state' ], $_GET[ 'wcs_stripe_code' ] );
+			if ( 'success' === $setting ) {
+				add_action( 'admin_notices', array( $this, 'connection_success_notice' ) );
```

Comment:
Could `wc_add_notice()` be used here, or is the styling different?

Reply:
Isn't that only for front-end notices?

Reply:
But perhaps `WC_Admin_Notices::add_notice()` might reduce some code duplication?

Reply:
Yeah, looks like `WC_Admin_Notices::add_custom_notice` is a good way to add a persistent/dismissible notice, but I'd wired up the current notice to only appear when redirected from stripe.com and not on subsequent page loads – open to feedback on revising that.